lib/sycamore/sycamore/evaluation/evaluate.py

```diff
@@ -0,0 +1,137 @@
+from typing import Dict, List, Union
+from abc import ABC, abstractmethod
+
+import os
+from sycamore.data import Element, Document
+from sycamore.evaluation.pipeline import EvaluationPipeline
+from sycamore.evaluation import EvaluationDataPoint
+from sycamore.evaluation.metrics import document_retrieval_metrics, rouge_metrics
+from sycamore import Context
+
+
+def add_searchContext_to_datapoint(datapoint: Document) -> List[Element]:
+    assert isinstance(datapoint, Document)
```

Comment:
Is this assert necessary?

Reply:
converted it to check .type is EvaluationDataPoint